src/web/jcc/client/components/TaskSettingsPanel.tsx

```diff
@@ -81,6 +84,58 @@ function TaskOptionsPanel(props: Props) {
         props.onChange(newTask)
     }
 
+    /**
+     * Checks if the Bottom dive is currently checked.
+     * 
+     * @returns {boolean} True if Bottom dive is checked, false otherwise.
+     */
+    function isBottomDiveChecked(){
+
+        if (dive?.bottom_dive === undefined) {
+            return false
+        }
+        return dive.bottom_dive
+    }
+
+    /**
+     * Handles the toggle action for the dive task.
+     * Also calls sets default values for diveparameters.
```

Comment:
This needs to be reworded: Also calls sets default values for diveparameters.

I actually think it can be removed. The top line is good enough.